lib/ibancom/resources/iban/validation.rb

```diff
@@ -29,7 +29,10 @@ def supported_schemes
 
         def supported_scheme?(code)
           scheme = supported_schemes.find { |supported| supported.code == code }
-          scheme&.supported?
+
+          return false if scheme.nil?
+
+          scheme.supported?
```

Comment:
```suggestion
          !!scheme.supported?
```

It is another way to write it but I recognize it is not as clear as the one chosen 👍🏻